gate-oauth2/gate-oauth2.gradle

```diff
@@ -2,6 +2,7 @@ dependencies {
   implementation project(":gate-core")
   implementation "com.netflix.spectator:spectator-api"
   implementation "io.spinnaker.fiat:fiat-api:$fiatVersion"
+  implementation "io.spinnaker.fiat:fiat-core:$fiatVersion"
```

Comment:
If this is a refactoring... why this change?

Reply:
[`Optional.ofNullable(Retrofit2SyncCall.execute(front50Service.getServiceAccounts()))
          .orElse(new ArrayList<>())
          .stream()
          .anyMatch(sa -> email.equalsIgnoreCase(sa.getName()));`](https://github.com/spinnaker/gate/pull/1871/files#diff-aa3aea41bbdec29ed8cd54a2d016a50d0195c432b5ea0e27b978e7b1e2fb3a71R206)

The above Java code depends on the ServiceAccount class, so I added the dependency.

Reply:
Yeah, without it, I get:
```
/Users/dbyron/src/spinnaker/gate/gate-oauth2/src/main/java/com/netflix/spinnaker/gate/security/oauth2/SpinnakerUserInfoTokenServices.java:206: error: cannot access ServiceAccount
      return Optional.ofNullable(Retrofit2SyncCall.execute(front50Service.getServiceAccounts()))
                                                  ^
  class file for com.netflix.spinnaker.fiat.model.resources.ServiceAccount not found
```